packages/compass-web/scripts/electron-proxy.js

```diff
@@ -118,12 +118,15 @@ class AtlasCloudAuthenticator {
    * @returns {Promise<string[]>}
    */
   async #getCloudSessionCookies() {
-    const cloudHostCookies = await session.defaultSession.cookies.get({
-      domain: CLOUD_CONFIG_VARIANTS === 'local' ? 'localhost' : 'mongodb.com',
-    });
-    return cloudHostCookies.map((cookie) => {
-      return `${cookie.name}=${cookie.value}`;
-    });
+    const tld = CLOUD_CONFIG_VARIANTS === 'local' ? 'localhost' : 'mongodb.com';
+    const cloudHostCookies = (await session.defaultSession.cookies.get({}))
+      .filter((cookie) => {
+        return cookie.domain?.endsWith(tld) ?? true;
```

Comment:
Ah TIL domain isn't always there https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/Cookies#define_where_cookies_are_sent

Reply:
Hah, yeah, I was mostly going off of types here, I think for proxy purposes it's easier to include the ones that don't have any value there just in case